### PR TITLE
31 accordion

### PR DESCRIPTION
**Pull request by bameyrick**
This could probably do with more documentation, design wise and technical implementation. I could do the technical but might not be best for the design and rationale documentation.

### CODE REVIEW

File: src/components/accordion/index.njk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+## When to use this component 
+This component can be used to condense large sections of related content on a page. It allows a user to quickly scan a collated group of content titles.
+
+## When not to use this component
+Accordions should not be used if the content inside the accordion is critial to the user to be able to progress with their task.
+
+{{
+    patternlibExample("components/accordion/examples/accordion/index.njk")
```
Comment (boxadesign) [inline src/components/accordion/index.njk]: Need to update to use `patternlibExample({"path":"components/accordion/examples/accordion/index.njk"})`

Comment (bameyrick) [reply src/components/accordion/index.njk]: Good catch 👍

**Comment by bameyrick** [comment]
[Accordion](https://trello.com/c/8aacxSBp/31-accordion)

**Comment by jrbarnes9** [comment]
It would be useful to explain when to use accordions over tabs, and vice versa.  Happy to push some more design/rationale content for this if you like.

**Comment by bameyrick** [comment]
> versa. Happy to push some more design/rationale content for this if you like.

That would be good!

**Pull request merged**